heracles-resources/backend/src/main/java/io/uhndata/cards/heracles/internal/export/ExportTask.java

```diff
@@ -99,6 +103,15 @@ public void run()
             }
         } catch (Exception e) {
             LOGGER.error("Failed to perform the nightly export", e.getMessage(), e);
+
+            // Create an nt:file node under /LoggedEvents/ storing the stack trace of this failure
```

Comment:
This is too technical.

```suggestion
            // Store the stack trace of this failure
```

Reply:
Fixed - https://github.com/data-team-uhn/cards/pull/1563/commits/69db72ea0a9afa9884193b478c76453ba238d80a :heavy_check_mark: